torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2468,7 +2513,7 @@ def _make_tensor_helper(shape, low=None, high=None):
            supports_tensor_out=False,
            skips=(
                SkipInfo('TestCommon', 'test_variant_consistency_jit',
-                        dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16)),
+                        dtypes=all_types_and_complex_and(torch.bool, torch.fl16, torch.bfloat16)),
```

Comment:
Accidental change? (that would broke the compilation)
```suggestion
                        dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16)),
```

Reply:
Thanks for pointing out, yes it is indeed accidental change :)